crates/biome_formatter/src/macros.rs

```diff
@@ -328,18 +328,28 @@ macro_rules! format {
 /// [`MostExpanded`]: crate::format_element::BestFittingElement::most_expanded
 #[macro_export]
 macro_rules! best_fitting {
-    ($least_expanded:expr, $($tail:expr),+ $(,)?) => {{
-        #[expect(clippy::macro_metavars_in_unsafe)]
-        unsafe {
-            $crate::BestFitting::from_arguments_unchecked($crate::format_args!($least_expanded, $($tail),+))
-        }
-    }}
+    ($least_expanded:expr, $($tail:expr),+ $(,)?) => {
+        // FIXME: Using any block in this macro causes a "temporary value dropped while borrowed"
+        //        (E0716) error since Rust 2024 edition. It seems temporary lifetime extension is
+        //        not working correctly, so it is a compiler bug?
+        $crate::macros::__macro_helper::best_fitting($crate::format_args!($least_expanded, $($tail),+))
+    };
```

Comment:
😓

Reply:
per https://discord.com/channels/1132231889290285117/1144208699540373565/1345665295309733961, it seems a tail expression so it isn't a bug. I don't think this is the best approach; any suggestions are welcome! 🙏